SmartDeviceLink/SDLProtocolListener.h

```diff
@@ -17,11 +17,6 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)handleProtocolEndServiceNAKMessage:(SDLProtocolMessage *)endServiceNAK;
 
 // Older protocol handlers
-- (void)handleProtocolStartSessionACK:(SDLServiceType)serviceType sessionID:(Byte)sessionID version:(Byte)version __deprecated_msg("use handleProtocolStartSessionACKMessage: instead");
```

Comment:
The `handleProtocolStartSessionACK` protocol handler is still being used in the `SDLProxy` class. The `SDLProxy` class needs to implement the `handleProtocolStartServiceACKMessage`protocol handler, otherwise setup will not complete.  This is why setup is not finishing when the protocol string is **com.smartdevicelink.prot2**.

Reply:
This needs to get fixed on **develop** as well.